src/components/TodoInput.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+function TodoInput({ todoList, setTodoList }) {
+  const [todo, setTodo] = useState('');
+  const onHandleInputChange = (e) => {
+    setTodo(e.target.value);
+  };
+  const onTodoButtonClick = () => {
+    const newTodo = {
+      id: Date.now(),
+      content: todo,
+      isDone: false,
+    };
+    //null값 입력 방지
+    if (todo) {
+      todoList = todoList.concat(newTodo);
+      setTodoList(todoList);
```

Comment:
```suggestion
      setTodoList([...todoList, newTodo]);
```

이렇게 작성하면 더 깔끔할 것 같습니다!